### ISSUE TITLE
SDK: Tracestate propagation

### ISSUE THREAD

**Issue by rghetia** [open-telemetry/opentelemetry-go#75]
Implement tracestate propagation.

blocked on #74

**Comment by rghetia** [comment]
In Java there is a [default implementation of Tracestate](https://github.com/open-telemetry/opentelemetry-java/blob/master/api/src/main/java/io/opentelemetry/trace/Tracestate.java) as a part of the api.
If same approach is taken here then #75 can be rolled into #74.

**Comment by jmacd** [comment]
We will postpone this until the OpenTelemetry specification addresses how to handle the W3C tracestate.

**Comment by rghetia** [comment]
@rghetia to check if this is needed.

**Comment by MrAlias** [comment]
Closing in favor of #391
